### PR TITLE
Added Testing Coverage

### PR DESCRIPTION
**Pull request by rebekahrudd**
This feature adds the command to run pytest and coverage. It also lowers the passing threshold of the test coverage to 50% so the build continues to pass while we run tests going forward

### CODE REVIEW

File: execexam/main.py [hunk 1]
```diff
@@ -86,6 +86,7 @@ def run(  # noqa: PLR0913, PLR0915
         litellm_thread.start()
     # add the project directory to the system path
     sys.path.append(str(project))
+    print("This is the correct execexam! You did it! You made it local. You are the best!")
```
Comment (CalebKendra) [inline execexam/main.py:89]: consider the rich command `console.print` instead of python `print`

**Comment by boulais01** [comment]
Hello @rebekahrudd, I'd like to recommend converting this PR to a draft for now on account of the unresolved comment and the failing build. If the failing build is expected, I also recommend specifying that in the PR description. Thank you!

**Comment by gkapfham** [comment]
Hello @rebekahrudd, can you please:

- Resolve the `print` statement in the code
- Convert the PR to a draft until you are finished with the final version

@CalebKendra and @boulais01 thanks for the feedback, I agree with all the points you shared!

File: execexam/main.py (hunk 1 above)
Comment (gkapfham) [reply execexam/main.py:99]: Yes, please note that we should not use standard `print` statements in the code. It is also very important to point out that _this specific print_ should not be in the final version of the program as it was only there for debugging purposes.

Also, @Coltin2121 please make sure that you use correct commit messages that follow the Conventional Commits standard.

**Comment by rebekahrudd** [comment]
That print line has actually been deleted in the fork we are using as a class. Here is the commit message where it was deleted: https://github.com/PCain02/execexam/commit/2093d054ef343423cba522e772e6f394ff0d4e28. Is it possible to add that commit to this PR?

**Comment by PCain02** [comment]
> That print line has actually been deleted in the fork we are using as a class. Here is the commit message where it was deleted: [PCain02@2093d05](https://github.com/PCain02/execexam/commit/2093d054ef343423cba522e772e6f394ff0d4e28). Is it possible to add that commit to this PR?

You should be able to sync your branch with the main branch with the sync button. Then deal with any merge conflicts and go from there. Hope this helps.

image

**Comment by gkapfham** [comment]
@hannahb09 can you please give us an update on whether or not you are going to be able to repair the build that continues to break for this pull request?

**Comment by gkapfham** [comment]
Hello @rebekahrudd, this PR has conflicts with specific files. Please also note that there are at least two other PRs that are all related to this one, at least in the name. As such, the other members of our team are not going to be clear as to which one should be merged first (or at all). Can you please investigate these issues and then report back?

**Comment by gkapfham** [comment]
Hello @rebekahrudd can we close this PR? Or, do you intend to push it forward in the future?

**Comment by rebekahrudd** [comment]
We can close this PR

**Comment by rebekahrudd** [comment]
We can close this PR

**Pull request closed without merging**